src/languages/es-ES.ts

```diff
@@ -1881,36 +1881,29 @@ export default class extends Language {
 		COMMAND_LEARN_EXTENDED: builder.display('learn', {
 			extendedHelp: `
 				Utiliza una búsqueda difusa para comparar también con coincidencias cercanas.
+				Los movimientos se dividen en cada \`,\` (espacio de coma)
+				y puede proporcionar tantos movimientos como desee.
```

Comment:
Given the English version:
```ts
Moves split on every \`, \` (comma space) and you can provide
as many moves as you wish.
```
Then this should be:
```suggestion
				Los movimientos se dividen en cada \`, \` (coma y espacio)
				y puedes proporcionar tantos movimientos como desee.
```